packages/textlint-scripts/package.json

```diff
@@ -33,6 +33,9 @@
         "url": "https://github.com/textlint/textlint/issues"
     },
     "homepage": "https://github.com/textlint/textlint/tree/master/packages/textlint-scripts",
+    "engines": {
+      "node": ">=18.19.0"
+    },
```

Comment:
textlint-scriptsr equire latest Node.js 18.x.
In the future, we might implement the hook processing using the `node:module` register API, so I'm choosing the latest version, 18.19.0.

- related? #1123